<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FtpClient</class>
 <widget class="QWidget" name="FtpClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FtpClient</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{font-family:&quot;微软雅黑&quot;;}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="background_widget" native="true">
     <property name="styleSheet">
      <string notr="true">#background_widget{background-color:#F8F8F8;}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="main_widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="top_widget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#top_widget{background-color:#0096ff;}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnMine">
              <property name="styleSheet">
               <string notr="true">QPushButton{background-color: rgba(255, 255, 255,0);background-position:center;background-repeat:no-repeat;border-style:solid; }
QPushButton:hover{background-color:#E33D29;}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../FtpClient.qrc">
                <normaloff>:/FtpClient/res/5.png</normaloff>:/FtpClient/res/5.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnClose">
              <property name="styleSheet">
               <string notr="true">QPushButton{background-color: rgba(255, 255, 255,0);background-position:center;background-repeat:no-repeat;border-style:solid; }
QPushButton:hover{background-color:#E33D29;}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../FtpClient.qrc">
                <normaloff>:/FtpClient/res/7.png</normaloff>:/FtpClient/res/7.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="operation_widget" native="true">
           <property name="styleSheet">
            <string notr="true">QPushButton{background: #e0eeff;border: 1px solid #3c91f3;border-radius: 2px;padding:5 10;color:#005bc3;}
QPushButton:hover{color:white;background-color:#0096FF;}
#operationWidget{background-color: white;}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="btnOpen">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>站点管理</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="AddPage">
            <property name="styleSheet">
             <string notr="true">#AddPage{background-color:#F8F8F8;}</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QToolButton" name="toolButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QToolButton{
background-color:white;
border-style:solid;
border-top-left-radius:12px;  
border-top-right-radius:12px;  
border:1px;  
border-radius:6px;
padding:0 0 0 0; 
}
QToolButton:hover {       
background:rgb(240, 240, 240);
border-top: 1px solid QLinearGradient(y0:0, y1:1,stop: 0 #DADADA, stop: 1 transparent);
border-left: 1px solid QLinearGradient(x0:0, x1:1,stop: 0 #DADADA, stop: 1 transparent);
border-bottom: 1px solid QLinearGradient(y0:-1, y1:1,stop: 0 transparent, stop: 1  #e3e3e3);
border-right: 1px solid QLinearGradient(x0:-1, x1:1,stop:  0 transparent, stop: 1 #e3e3e3);
                                  
}
</string>
               </property>
               <property name="text">
                <string>拖动文件至此或点击</string>
               </property>
               <property name="icon">
                <iconset resource="../FtpClient.qrc">
                 <normaloff>:/FtpClient/res/fileUpload.png</normaloff>:/FtpClient/res/fileUpload.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>84</width>
                 <height>84</height>
                </size>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="autoRaise">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="ProgressPage">
            <property name="styleSheet">
             <string notr="true">#ProgressPage{background-color:#F8F8F8;}</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QProgressBar" name="progressBar">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
}
QProgressBar::chunk {
    background-color: #05B8CC; 
    width: 10px;             
    margin: 0.5px;             
}</string>
               </property>
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../FtpClient.qrc"/>
 </resources>
 <connections/>
</ui>
